examples/.gitignore

```diff
@@ -0,0 +1,31 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
This file should be removed

Reply:
This file was required for the script. If the script is not required, I shall remove this file too.

Reply:
The script should rely on the template's `.gitignore`.